app/ergoCubEmotions/conf/config.ini

```diff
@@ -2,30 +2,25 @@
 num_expressions 4
 num_transitions 12
 
-# [expression_0]
-# name neutral
-# type image
-# file expressions/images/exp_img_0.png
-
-# [expression_1]
-# name happy
-# type image
-# file expressions/images/exp_img_1.png
-
-# [expression_2]
-# name alert
-# type image
-# file expressions/images/exp_img_2.png
-
-# [expression_3]
-# name shy
-# type image
-# file expressions/images/exp_img_3.png
-
-# [expression_4]
-# name wink
-# type video
-# file expressions/videos/exp_vid_0.mp4
+#[expression_0]
+#name neutral
+#type image
+#file expressions/images/exp_img_0.png
+#
+#[expression_1]
+#name happy
+#type image
+#file expressions/images/exp_img_1.png
+#
+#[expression_2]
+#name alert
+#type image
+#file expressions/images/exp_img_2.png
+#
+#[expression_3]
+#name shy
+#type image
+#file expressions/images/exp_img_3.png
```

Comment:
All these commented lines are not used? can we get rid of them?

Reply:
I kept them to have an example of how to handle images instead of videos. So far, we haven't used them, but if we need them, we will just uncomment them. If you think that they're not necessary, I'll remove those lines